### DISCUSSION TITLE
Cha-ching! Found a block - Notification not exactly accurate.

### DISCUSSION THREAD

**Discussion by k2skaterii** [guydavis/machinaris#386] [General]
My Birthday isn't till next week, looks like Chia sent my present a little early.  I've found 3 blocks, 2 in the last 10 days.  No complaints there.

When I got up today my phone let me know that I had found a block.  Cool.  I logged into my Machinaris GUI and got a little surprise in my Notifications.  There were 2x Cha-chings.  Turns out only one of them was valid.  Looks like 1 of them was the payout for finding the block and the other one was for the pool's share of the block I found.  

I did check my wallet and I can confirm I did not receive the 1.75 XCH payout.

Honestly this isn't a huge issue, but I figured I'd share either way!

![image](https://user-images.githubusercontent.com/87842648/139687782-f3f474fa-37d4-4f68-ab3c-78cbcbb24122.png)

**Comment by guydavis** [comment]
Good day!  Thanks for the note.  Yes, this looks to be a [known issue](https://github.com/martomi/chiadog/issues/286) in Chiadog.  Apologies for the confusion.
